packages/SwingSet/misc-tools/store-stats/src/store-stats.js

```diff
@@ -0,0 +1,160 @@
+import { StatsD } from 'hot-shots';
+
+const dogstatsd = new StatsD({
+  errorHandler(error) {
+    console.log('StatsD socket errors: ', error);
+  },
+});
+
+/**
+ * @param {import('better-sqlite3').Database} db
+ */
+export const makeSwingstore = db => {
```

Comment:
let's name this slightly differently, to avoid confusion with the `makeSwingStore` defined in `@agoric/swing-store`.. how about `makeDB` ?